class.slide

```diff
@@ -168,13 +162,29 @@ Go Overview
 - Explicitly handle the error: log it, ignore it, or pass it upward.
 - By convention, return error as last item in return tuple.  Gets away from using out-of-band values for errors.
 - Use `nil` when there is no error.
+- Errors are values, you can do clever things with them.
 
 * Error Example
 .play -edit err.go /START MAIN OMIT/,/END MAIN OMIT/
 
 * Error Example
 .play -edit err.go /START FUNC OMIT/,/END FUNC OMIT/
 
+* Error Example - Wrapping
+.play -edit err2.go /START MAIN OMIT/,/END MAIN OMIT/
+
+* Documentation
+- Docs are never far from the source.
+- You can do the same.
+- By convention, start doc comments with the name of the function/type.
+.link https://golang.org/pkg/crypto/tls/#X509KeyPair LoadX509KeyPair() Example
```

Comment:
This was way later... but I thought it would be neater up here

Reply:
This is a good idea.  I might add a link to DevDocs as well.